src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3501,7 +3501,7 @@ private static void AddProviderEnumKind(ManifestBuilder manifest, FieldInfo stat
 #endif
             return;
             Error:
-            manifest.ManifestError(SR.Format(SR.EventSource_EnumKindMismatch, staticField.Name, staticField.FieldType.Name, providerEnumKind));
+            manifest.ManifestError(SR.Format(SR.EventSource_EnumKindMismatch, staticField.FieldType.Name, providerEnumKind));
```

Comment:
@dotnet/area-system-diagnostics-tracesource could you please review the changes in /Tracing/ ? It's not clear to me which 2 of the 3 parameters are supposed to go into the string. Today it's implicitly the first 2.

Reply:
@brianrob can you page in enough context briefly to confirm that the two changes in this file are correct? The 2nd one just preserves existing behavior, although that might be wrong.

After that this is good to merge.

Reply:
Yup, the changes in `EventSource.cs` look good to me.  It looks like there were additional parameters passed to `SR.Format` that should never have been there.

Reply:
thanks @brianrob !